<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="DataGridViewDataErrorEventArgs.xml" source-language="en-US" target-language="pl-PL">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac507424dd07417db23cdbec88dd22f5097f25643bc.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">07424dd07417db23cdbec88dd22f5097f25643bc</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Provides data for the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> event.</source>
          <target state="translated">Udostępnia dane dla <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> zdarzeń.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Handling the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event lets you handle exceptions thrown by code outside your control (for example, by an external data source).</source>
          <target state="translated">Obsługa <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> zdarzenie pozwala obsługi wyjątków zgłaszanych przez kod poza formantu (na przykład przez zewnętrznego źródła danych).</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Use the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> property to determine the state of the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> at the time of the exception.</source>
          <target state="translated">Użyj <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> właściwości w celu określenia stanu <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> w czasie wystąpienia wyjątku.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Use the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property to retrieve the exception data.</source>
          <target state="translated">Użyj <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> właściwość, aby pobrać Dane wyjątku.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>If you want to handle the exception by additional event handlers, set the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to <ph id="ph2">`true`</ph>.</source>
          <target state="translated">Do obsługi wyjątku przez programy obsługi zdarzeń dodatkowe, należy ustawić <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> właściwości <ph id="ph2">`true`</ph>.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>The <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> and <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A&gt;</ph> properties normally indicate the cell in which the data error occurred.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> i <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A&gt;</ph> właściwości zwykle wskazują komórki, w którym wystąpił błąd danych.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>When the error occurs in an external data source, however, the data source may not provide the column in which the error occurred.</source>
          <target state="translated">Po wystąpieniu błędu w zewnętrznym źródłem danych, jednak źródła danych może nie zawiera kolumny, w którym wystąpił błąd.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>In this case, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> property typically indicates the column of the current cell at the time of the error.</source>
          <target state="translated">W takim przypadku <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> właściwość zwykle wskazuje kolumny bieżącej komórki w chwili wystąpienia błędu.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>The following code example demonstrates how to respond to information provided by the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs&gt;</ph> class.</source>
          <target state="translated">Poniższy przykład kodu pokazuje, jak odpowiedzieć na informacje o <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs&gt;</ph> klasy.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Ten przykład jest częścią większego przykładu dostępne w <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> temat klasy.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The exception that occurred.</source>
          <target state="translated">Wyjątek, który wystąpił.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The column index of the cell that raised the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</source>
          <target state="translated">Indeks kolumny komórki, która wywołała <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The row index of the cell that raised the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</source>
          <target state="translated">Indeks wiersza komórki, która wywołała <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>A bitwise combination of <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> values indicating the context in which the error occurred.</source>
          <target state="translated">Bitowe połączenie <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> wartości, wskazując kontekst, w którym wystąpił błąd.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventArgs" /&gt;</ph> class.</source>
          <target state="translated">Inicjuje nowe wystąpienie klasy <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventArgs" /&gt;</ph> klasy.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>Gets details about the state of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridView" /&gt;</ph> when the error occurred.</source>
          <target state="translated">Pobiera szczegóły dotyczące stanu <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridView" /&gt;</ph> wystąpienia błędu.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>A bitwise combination of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> values that specifies the context in which the error occurred.</source>
          <target state="translated">Bitowe połączenie <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> wartości, które określa kontekst, w którym wystąpił błąd.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>Members of the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts&gt;</ph> enumeration may be combined using the bitwise <ph id="ph2">`OR`</ph> operator to represent the state of a data-bound <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> when a data error occurred.</source>
          <target state="translated">Elementy członkowskie <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts&gt;</ph> wyliczenia mogą być łączone przy użyciu operatora testu koniunkcji <ph id="ph2">`OR`</ph> operator do reprezentowania stanu z danymi <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> gdy wystąpił błąd danych.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>For example, if a user enters an invalid cell value (such as entering a name in a cell that requires a date) and then selects a different cell, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> will try to commit the invalid cell value.</source>
          <target state="translated">Na przykład, jeśli użytkownik wprowadzi wartość komórki nieprawidłowe (na przykład w komórce, która wymaga datę, wprowadzając nazwę), a następnie wybiera inną komórkę <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> spróbuje przekazać wartość komórki nieprawidłowy.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>When the commit fails, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> will raise a <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event whose <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> property will have a value of <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.Commit&gt;</ph> and <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.CurrentCellChange&gt;</ph>.</source>
          <target state="translated">W przypadku awarii zatwierdzenia <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> zgłosi <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> zdarzeń których <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> właściwość będzie mieć wartość <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.Commit&gt;</ph> i <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.CurrentCellChange&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>The following code example demonstrates how to investigate the error context.</source>
          <target state="translated">Poniższy przykład kodu pokazuje, jak do sprawdzania, czy kontekst błędu.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Ten przykład jest częścią większego przykładu dostępne w <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> temat klasy.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>Gets the exception that represents the error.</source>
          <target state="translated">Pobiera wyjątek, który reprezentuje błąd.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>An <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that represents the error.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> Reprezentujący błąd.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>To handle the exception by more than one <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>, throw the exception again at the end of a handler by setting the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to <ph id="ph3">`true`</ph>.</source>
          <target state="translated">Do obsługi wyjątku przez więcej niż jedną <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>, zgłosić wyjątek ponownie po zakończeniu obsługi przez ustawienie <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> właściwości <ph id="ph3">`true`</ph>.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>The following code example demonstrates checking the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property.</source>
          <target state="translated">Poniższy przykład kodu pokazuje sprawdzanie <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> właściwości.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Ten przykład jest częścią większego przykładu dostępne w <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> temat klasy.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>Gets or sets a value indicating whether to throw the exception after the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventHandler" /&gt;</ph> delegate is finished with it.</source>
          <target state="translated">Pobiera lub ustawia wartość wskazującą, czy ma zostać zgłoszony wyjątek po <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventHandler" /&gt;</ph> zakończeniu delegata działania z nim.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source><ph id="ph1">&lt;see langword="true" /&gt;</ph> if the exception should be thrown; otherwise, <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;see langword="true" /&gt;</ph> Jeśli wyjątek powinien zostać zgłoszony; w przeciwnym razie <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>The default is <ph id="ph1">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated">Wartość domyślna to <ph id="ph1">&lt;see langword="false" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>Set this property to <ph id="ph1">`false`</ph> if the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> has dealt with the exception represented by the <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property, and you do not want to propagate the exception to another <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> that may handle the <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event.</source>
          <target state="translated">Ustaw tą właściwość na <ph id="ph1">`false`</ph> Jeśli <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> prowadzonych wyjątek reprezentowany przez <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> właściwości, a nie chcesz propagowanie wyjątku do innego <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> która może obsłużyć <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> zdarzeń.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>For debugging purposes, it can be useful to propagate the exception and inspect the stack trace for information regarding the context of the error.</source>
          <target state="translated">Na potrzeby debugowania, może być przydatne do propagowanie wyjątku i przejrzyj ślad stosu, aby uzyskać informacje dotyczące kontekstu błędu.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>The following code example demonstrates using the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to indicate that the exception should not be thrown.</source>
          <target state="translated">Poniższy przykład kodu pokazuje, przy użyciu <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> Właściwość wskazująca, czy wyjątek nie powinien zostać zgłoszony.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Ten przykład jest częścią większego przykładu dostępne w <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> temat klasy.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>When setting this property to <ph id="ph1">&lt;see langword="true" /&gt;</ph>, the <ph id="ph2">&lt;see cref="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception" /&gt;</ph> property value is <ph id="ph3">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated">Gdy ustawienie dla tej właściwości <ph id="ph1">&lt;see langword="true" /&gt;</ph>, <ph id="ph2">&lt;see cref="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception" /&gt;</ph> wartość właściwości jest <ph id="ph3">&lt;see langword="null" /&gt;</ph>.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>